src/error.rs

```diff
@@ -85,6 +85,14 @@ pub enum Error {
 
     #[error("Cannot convert quantity [{value}] to Java heap value with unit [{target_unit}].")]
     CannotConvertToJavaHeapValue { value: String, target_unit: String },
+    #[error("Conversion error: [message]")]
+    ConversionError { message: String },
```

Comment:
This isn't used anywhere, is it?

Reply:
No. It's not in `main` anymore. Rebased on top of main & removed.